.ocamlformat

```diff
@@ -0,0 +1,2 @@
+version=0.13.0
+profile=conventional
```

Comment:
you can even leave this line out as it's now the default

Reply:
Doesn't it make more sense to explicitly declare it? If the default changes this won't break.

I don't really mind, in particular since we set the version so it's unlikely that anything would break anyway!

Reply:
Agreed it doesn't hurt to explicitly set it to `conventional` (I do the same), I think we can also set `parse-docstrings=true` as it is not the default yet.